### ISSUE TITLE
bug: @nrwl/nest:library - redundant node files

### ISSUE THREAD

**Issue by MaciejSikorski** [nrwl/nx#2942]
# Prerequisites

- [x] I am running the latest version
- [x] I checked the documentation (nx.dev) and found no answer
- [x] I checked to make sure that this issue has not already been filed
- [x] I'm reporting the issue to the correct repository (not related to React, Angular or any dependency)

## Expected Behavior
My workspace should use one lint system.
The generated library should have only nestjs and the ecosystem files.

## Current Behavior
When I create a library with @nrwl/nest:library it creates a the-library-name.ts file in the library which is redundant for the Nestjs.
 
## Failure Information (for bugs)

Please help provide information about the failure if this is a bug. If it is not a bug, please remove the rest of this template.

### Steps to Reproduce

Please provide detailed steps for reproducing the issue.

1. npx create-nx-workspace@latest repro-workspace
2. Choose an empty workspace
3. npm install -D @nrwl/nest
4. nx generate @nrwl/nest:library --name=ui-rest --directory=filesystem --controller --tags='scope:filesystem, type:ui' --target=es2018
5. Notice libs/filesystem/ui-rest/src/lib/filesystem-ui-rest.ts file

If you can provide steps to reproduce from scratch, that would be enormously appreciated (i.e. where the first step is `npx create-nx-workspace@latest repro-workspace`)

### Context

Please provide any relevant information about your setup:

- version of Nx used 
```
  @nrwl/angular : Not Found
  @nrwl/cli : 9.2.3
  @nrwl/cypress : Not Found
  @nrwl/eslint-plugin-nx : 9.2.3
  @nrwl/express : Not Found
  @nrwl/jest : 9.2.3
  @nrwl/linter : 9.2.3
  @nrwl/nest : 9.2.3
  @nrwl/next : Not Found
  @nrwl/node : 9.2.3
  @nrwl/react : Not Found
  @nrwl/schematics : Not Found
  @nrwl/tao : 9.2.3
  @nrwl/web : Not Found
  @nrwl/workspace : 9.2.3
  typescript : 3.8.3
```
- 3rd-party libraries and their versions
```
@nestjs/core: 6.11.11
@nestjs/common: 6.11.11
```
- and most importantly - a use-case that fails
`Executing task: nx generate @nrwl/nest:library --name=ui-rest --directory=filesystem --controller --tags='scope:filesystem, type:ui' --target=es2018`

**Comment by beeman** [comment]
This should be fixed in #2920

**Comment by FrozenPandaz** [comment]
Fixed in #2920!
